### DISCUSSION TITLE
Chrome flags exploits

### DISCUSSION THREAD

**Discussion by Luigi-Master** [3kh0/ext-remover#1360] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
are there any exploits I can do using chromeflags?

**Comment by S-PScripts** [comment]
https://github.com/search?q=repo%3AS-PScripts%2Fchromebook-utilities+chrome%3A%2F%2Fflags&type=code

**Reply by S-PScripts** [reply]
you can find some here
